docker/ci/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM rust:1.53-slim
+RUN apt-get update && apt-get install -y pkg-config libssl-dev && cargo install cargo-audit
```

Comment:
There ought to be an `apt-get clean` in the same layer step, like [here](https://github.com/input-output-hk/rust-ci-docker/blob/0e678de1fe98f23455262bc44e737b4299f0001e/stable/Dockerfile#L8).

Conversely, `cargo install` might be done in a different later `RUN` step from the apt-get installs; this way layer caching is more affine to the respective package source's changes.

Reply:
```suggestion
RUN apt-get update && apt-get install -y pkg-config libssl-dev && apt-get clean
RUN cargo install cargo-audit
```